src/helpers/requestManager.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
```

Comment:
I am not really feeling the use of a helper directory for this. Maybe `src/lib` would be better suited 💭